pkgs/development/python-modules/mitmproxy/default.nix

```diff
@@ -113,10 +114,12 @@ buildPythonPackage rec {
     "test_flowview"
     # ValueError: Exceeds the limit (4300) for integer string conversion
     "test_roundtrip_big_integer"
-
     "test_wireguard"
     "test_commands_exist"
     "test_statusbar"
+    # AssertionError: Playbook mismatch!
+    "test_untrusted_cert"
+    "test_mitmproxy_ca_is_untrusted"
```

Comment:
looks like those 2 indeed fail in nixpkgs, only on small error test/exception mismatches (different dependencies from upstream?). I guess fair enough to exclude more as long as enough positive tests are still present